packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -356,6 +358,53 @@ class VifStatus extends BaseComponent {
   }
 }
 
+const AdvancedSettingsModal = decorate([
+  provideState({
+    effects: {
+      toggleTxChecksumming() {
+        const { onChange, value } = this.props
+        onChange({
+          ...value,
+          txChecksumming: !value.txChecksumming,
+        })
+      },
+    },
+  }),
+  injectState,
+  ({ effects, state, value }) => (
+    <Container>
+      <SingleLineRow>
+        <Col mediumSize={4}>
+          <strong>{_('txChecksumming')}</strong>
+        </Col>
+        <Col mediumSize={8}>
+          <StateButton
+            disabledLabel={_('off')}
+            enabledLabel={_('on')}
+            handler={effects.toggleTxChecksumming}
+            state={value.txChecksumming}
+          />
+        </Col>
```

Comment:
`6`/`6` `Col`s and the `StateButton` could be pulled to the right to fill the modal better.